hardware.inc

```diff
@@ -442,57 +442,65 @@ def AUD3WAVE_SIZE equ 16
 ; PPU graphics control
 def rLCDC equ $FF40
 
-def LCDCB_ON      equ 7 ; whether the PPU (and LCD) are turned on          [r/w]
-def LCDCB_WIN9C00 equ 6 ; which tilemap the Window reads from              [r/w]
-def LCDCB_WINON   equ 5 ; whether the Window is enabled                    [r/w]
-def LCDCB_BLKS    equ 4 ; which "tile blocks" the BG and Window use        [r/w]
-def LCDCB_BG9C00  equ 3 ; which tilemap the BG reads from                  [r/w]
-def LCDCB_OBJ16   equ 2 ; how many pixels tall each OBJ is                 [r/w]
-def LCDCB_OBJON   equ 1 ; whether OBJs are enabled                         [r/w]
-def LCDCB_BGON    equ 0 ; (DMG only) whether the BG is enabled             [r/w]
-def LCDCB_PRION   equ 0 ; (CGB only) whether OBJ priority bits are enabled [r/w]
-    def LCDCF_OFF     equ 0 << LCDCB_ON
-    def LCDCF_ON      equ 1 << LCDCB_ON
-    def LCDCF_WIN9800 equ 0 << LCDCB_WIN9C00
-    def LCDCF_WIN9C00 equ 1 << LCDCB_WIN9C00
-    def LCDCF_WINOFF  equ 0 << LCDCB_WINON
-    def LCDCF_WINON   equ 1 << LCDCB_WINON
-    def LCDCF_BLKS    equ 1 << LCDCB_BLKS
-        def LCDCF_BLK21 equ 0 << LCDCB_BLKS
-        def LCDCF_BLK01 equ 1 << LCDCB_BLKS
-    def LCDCF_BG9800  equ 0 << LCDCB_BG9C00
-    def LCDCF_BG9C00  equ 1 << LCDCB_BG9C00
-    def LCDCF_OBJ8    equ 0 << LCDCB_OBJ16
-    def LCDCF_OBJ16   equ 1 << LCDCB_OBJ16
-    def LCDCF_OBJOFF  equ 0 << LCDCB_OBJON
-    def LCDCF_OBJON   equ 1 << LCDCB_OBJON
-    def LCDCF_BGOFF   equ 0 << LCDCB_BGON
-    def LCDCF_BGON    equ 1 << LCDCB_BGON
-    def LCDCF_PRIOFF  equ 0 << LCDCB_PRION
-    def LCDCF_PRION   equ 1 << LCDCB_PRION
+def B_LCDC_ENABLE   equ 7 ; whether the PPU (and LCD) are turned on          [r/w]
+def B_LCDC_WIN_MAP  equ 6 ; which tilemap the Window reads from              [r/w]
+def B_LCDC_WINDOW   equ 5 ; whether the Window is enabled                    [r/w]
+def B_LCDC_BLOCKS   equ 4 ; which "tile blocks" the BG and Window use        [r/w]
+def B_LCDC_BG_MAP   equ 3 ; which tilemap the BG reads from                  [r/w]
+def B_LCDC_OBJ_SIZE equ 2 ; how many pixels tall each OBJ is                 [r/w]
+def B_LCDC_OBJS     equ 1 ; whether OBJs are enabled                         [r/w]
+def B_LCDC_BG       equ 0 ; (DMG only) whether the BG is enabled             [r/w]
+def B_LCDC_PRIORITY equ 0 ; (CGB only) whether OBJ priority bits are enabled [r/w]
+    def LCDC_ENABLE   equ 1 << B_LCDC_ENABLE
+        def LCDC_OFF equ 0 << B_LCDC_ENABLE
+        def LCDC_ON  equ 1 << B_LCDC_ENABLE
+    def LCDC_WIN_MAP  equ 1 << B_LCDC_WIN_MAP
+        def LCDC_WIN_9800 equ 0 << B_LCDC_WIN_MAP
+        def LCDC_WIN_9C00 equ 1 << B_LCDC_WIN_MAP
```

Comment:
I like being able to explicitly write 9800 or 9C00, instead of just implying 9800 via the absence of 9C00